assets/js/translator.js

```diff
@@ -1307,7 +1307,7 @@ function setDefaultSrcLang() {
 
     // Convert language code from browser -> apertium format.
     function changeLangCode(lang) {
-        return lang.replace('-', '_');
+        return lang.split('-')[0];
```

Comment:
Why did this get switched back again? I thought we discussed that we need to preserve the variant since Apertium supports them.

Reply:
Replied below.